clap_generate/src/generators/shells/zsh.rs

```diff
@@ -335,29 +335,30 @@ fn get_args_of(p: &App) -> String {
 
 // Uses either `possible_vals` or `value_hint` to give hints about possible argument values
 fn value_completion(arg: &Arg) -> String {
-    if let Some(ref values) = arg.possible_vals {
+    if let Some(values) = &arg.possible_vals {
         format!(
             "({})",
             values
                 .iter()
-                .map(|v| escape_value(*v))
-                .collect::<Vec<String>>()
+                .map(|&v| escape_value(v))
+                .collect::<Vec<_>>()
                 .join(" ")
         )
     } else {
         match arg.value_hint {
+            ValueHint::AnyPath => "_files",
             ValueHint::FilePath => "_files",
```

Comment:
You don't need to define these since you have a default

Reply:
I'd rather keep it explicit.

Reply:
I'm with CreepySkeleton, I made this `match{}` exhaustive as well.